homeassistant/components/sensor/github.py

```diff
@@ -0,0 +1,215 @@
+"""
+Support for GitHub.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.github/
+"""
+from datetime import timedelta
+import logging
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    ATTR_NAME, CONF_ACCESS_TOKEN, CONF_NAME, CONF_PATH, CONF_URL)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['PyGithub==1.43.5']
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_REPOS = 'repositories'
+
+ATTR_LAST_COMMIT_MESSAGE = 'last_commit_message'
+ATTR_LAST_COMMIT_SHA = 'last_commit_sha'
+ATTR_LATEST_RELEASE_URL = 'latest_release_url'
+ATTR_OPEN_ISSUE_LATEST = 'open_issue_latest'
+ATTR_OPEN_ISSUES = 'open_issues'
+ATTR_OPEN_PULL_REQUEST_LATEST = 'open_pull_request_latest'
+ATTR_OPEN_PULL_REQUESTS = 'open_pull_requests'
+ATTR_PATH = 'path'
+ATTR_STARGAZERS = 'stargazers'
```

Comment:
Please consider the consistency of these keys, `last` vs. `latest` and prefix vs. postfix.

Reply:
Good point. I've switched last to latest:

![image](https://user-images.githubusercontent.com/28114703/52376453-13419880-2a5a-11e9-89df-a47640855f9c.png)

Reply:
Just checking that you do not want e.g. "latest open issue url" (to match e.g. "latest release url")?

I am thinking that consistency would make it easier to remember when making automations/templates but maybe there are reasons for doing it like it is now :)

Reply:
Good point. I've added one last commit 😃